coalib/bearlib/aspects/Smell.py

```diff
@@ -101,13 +100,32 @@ def func(a, b, c, d, e, f, g, h, i, j, k, l, m, n, o, p, q, r, s, t, z):
         parameters (maybe by creating more functions or using libraries).
         """
 
+
+@MethodSmell.subaspect
+class MethodLength:
+    """
+    This aspect checks for methods and functions with a large number
+    of lines.
+    """
+    class docs:
+        example = """
+            # Here is a method with one line of code
+            def func():
+                print('coala is always written with lowercase `c`.')
+        """
+        example_language = 'python'
+        importance_reason = """
+        The longer a method or function is, the harder it becomes to
+        understand and maintain it. Also long methods easily introduce
+        unwanted duplicated code, and bugs to our codebase.
+        """
+        fix_suggestions = """
+        Extracting methods or functions is a fix for this.
```

Comment:
Refactoring methods into smaller more generic methods. Making code more compact by using inline and language specific syntax tricks etc.